.terraform.lock.hcl

```diff
@@ -3,7 +3,7 @@
 
 provider "registry.terraform.io/hashicorp/aws" {
   version     = "3.23.0"
-  constraints = ">= 3.3.0, >= 3.20.0"
+  constraints = ">= 3.20.0, >= 3.22.0"
```

Comment:
The constraints for all providers are specified in the versions.tf file. 
The AWS constraint cannot be set to "=" otherwise when you run terraform apply you'll get an error, because some components of the cluster require different versions (I tried it).